mk/update-travis-yml.py

```diff
@@ -121,6 +121,11 @@ def format_entries():
             %(sources)s"""
 
 def format_entry(os, target, compiler, rust, mode):
+    if os == "linux" and compiler == "gcc-5" and rust == "nightly" and mode == "DEBUG":
```

Comment:
> I picked nightly because passing -C link-dead-code to rustc using RUSTFLAGS currently only works on beta and nightly. I can change to stable and it will start working when the next release happens. Until then coverage tracing won't be as accurate.

It's good to leave it as you have it. But, please add a comment with the explanation of why.



> I picked gcc-5 as an example. I wanted to run kcov on just one build. Should I do it on all of them?

> I picked gcc-5 as an example. I wanted to run kcov on just one build. Should I do it on all of them?

No, it is fine to just use gcc-5. But, please add a comment saying that it is restricted to one build for efficiency reasons only, and gcc-5 was chosen somewhat arbitrarily.

> The debug symbols are needed for coverage tracking.

Just add a comment explaining that coverage is run on debug builds because kcov requires debug symbols.

> Should I change the release builds to add them too on this pr?

It would be awesome if you did this, but perhaps a separate PR?